.github/workflows/tests.yaml

```diff
@@ -153,7 +150,7 @@ jobs:
           sudo apt install -y libapt-pkg-dev intltool fuse-overlayfs python3-apt
       - name: Install general python dependencies
         run: |
-          pip install python-apt@https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/python-apt/2.0.1ubuntu0.20.04.1/python-apt_2.0.1ubuntu0.20.04.1.tar.xz
+          pip install python-apt@https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/python-apt/2.4.0ubuntu1/python-apt_2.4.0ubuntu1.tar.xz
```

Comment:
I'm surprised this (the old one) even worked with noble!

Reply:
Me too!